lib/pf/Authentication/CreateLocalAccountRole.pm

```diff
@@ -15,6 +15,8 @@ use pf::Authentication::constants qw($LOCAL_ACCOUNT_UNLIMITED_LOGINS);
 
 has 'create_local_account' => (isa => 'Str', is => 'rw', default => 'no');
 has 'local_account_logins' => (isa => 'Str', is => 'rw', default => $LOCAL_ACCOUNT_UNLIMITED_LOGINS);
+has 'hash_passwords'       => (isa => 'Maybe[Str]', is => 'rw', default => 'ntlm');
```

Comment:
`hash_passwords=bcrypt` in `pf.conf.defaults`, we need to do same thing for sources.

Reply:
@nqb's comment sounds legit

Reply:
@julsemaan: not sure because `bcrypt` can't be use for 802.1X local auth.

Reply:
Then we should see if we change our default in pf.conf.defaults, but IMO, the default for the sources should be the same default as for the system